ocs/agents/registry/agent.py

```diff
@@ -146,8 +143,9 @@ def _publish_agent_ops(self, reg_agent):
             except ValueError as e:
                 self.log.warn(f"Improper field name: {field}\n{e}")
                 continue
-            msg['data'][field] = op_code
-        if msg['data']:
+            msg = {'block_name': field,
+                   'timestamp': time.time(),
+                   'data': dict(field=op_code)}
```

Comment:
This use of `dict()` results in `'data': {'field': 3}` (if `op_code==3`, for instance). Down the line this changes the field name that should be 
```
'observatory.registry.feeds.agent_operations.observatory_aggregator_record'
```
into
```
'observatory.registry.feeds.agent_operations.field'
```
If you want to use the `dict()` constructor you need to pass a list of tuples, i.e. `'data': dict([(field, op_code)])`.

Or you can use `'data': {field: op_code}`, which would be my preference.

Reply:
This is a good catch, thanks! I'm totally fine with switching to the regular dict format.